api/src/docker/README.md

```diff
@@ -0,0 +1,22 @@
+# Serverless
+
+We're using the serverless from work for deployment
+
+```
+sls deploy
+```
+But Kurt Hutten credentials for deployment, though if you wanted to set your own account you could deploy to that if you wanted to test.
+
+## testing changes locally
+
+Honestly it's a pain in the neck, for a number of reasons.
+
+You'll need to have Docker installed
+
+Because of the way the docker containers to be deployed as lambdas on aws are somewhat specialised for the purpose we're using `docker-compose` to spin one up for each function/endpoint. But more work needs to be done to get this to work with the app locally.
+
+```bash
+docker-compose up --build
```

Comment:
Because the lambda functions need are suposed to run in aws-land they can't be used directly, so I plan on making a dev only docker container that does a bit of aws emulation so that we can use them. I start doing with with the docker-compose file, but I think it will make more sense to get this bit of work done when I start actually iterating on the openscad api (atm it's very simple).

It's not great to be writing dev only code (emulating aws), and the serverless framework I'm using does have a offline plugin that I've used before, but doesn't seem to like docker lambdas.